test/unit_tests/molecules/Multiselect-spec.js

```diff
@@ -37,17 +53,89 @@ describe( 'Multiselect', function() {
+        option.defaultSelected = true;
+        multiselect.init();
+        var choices = document.querySelectorAll( '.cf-multi-select_choices li' );
+
+        expect( choices.length ).to.equal( 1 );
+        expect( choices[0].innerHTML ).to.contain( 'Debt collection' );
+      }
+    );
+
+    it( 'should log a helpful tip if passed a bad option value', function() {
+      var option = document.querySelector( 'option' );
+      option.value = 'Foo\'';
+      multiselect.init();
+      selectDom = document.querySelectorAll( 'select[multiple]' );
+      multiselectDom = document.querySelectorAll( '.cf-multi-select' );
+
+      expect( selectDom.length ).to.equal( 1 );
+      expect( multiselectDom.length ).to.equal( 0 );
+      sinon.assert.calledOnce( console.log );
+      sinon.assert.calledWithExactly(
+        console.log, '\'Foo\'\' is not a valid value'
+      );
+    } );
+  } );
+
+  describe( 'public methods', function() {
+    it( 'should open when the expand method is called', function() {
+      multiselect.init();
+      multiselect.expand();
+      multiselectDom = document.querySelector( '.cf-multi-select' );
+      var fieldset = multiselectDom.querySelector( '.cf-multi-select_fieldset' );
+
+      expect( multiselectDom.className ).to.equal( 'cf-multi-select active' );
+      expect( fieldset.getAttribute( 'aria-hidden' ) ).to.equal( 'false' );
+    } );
+
+    it( 'should close when the collapse method is called', function() {
+      multiselect.init();
+      multiselect.expand();
+      multiselect.collapse();
+      multiselectDom = document.querySelector( '.cf-multi-select' );
+      var fieldset = multiselectDom.querySelector( '.cf-multi-select_fieldset' );
+
+      expect( multiselectDom.className ).to.equal( 'cf-multi-select' );
+      expect( fieldset.getAttribute( 'aria-hidden' ) ).to.equal( 'true' );
+    } );
   } );
 
-  describe( 'dom creation', function() {
-    xit( 'should autocheck any selected options ' +
-         '(form submitted pages)', function() {
-      var options = selectDom.querySelectorAll( 'option' );
-      options[0].selected = 'selected';
+  describe( 'interactions', function() {
+    xit( 'should open when the search input is clicked', function() {
+      multiselect.init();
+      multiselectDom = document.querySelector( '.cf-multi-select' );
+      var fieldset = multiselectDom.querySelector( '.cf-multi-select_fieldset' );
+      var search = document.querySelector( '#test-select' );
+      search.click();
+
+      expect( document.activeElement.id ).to.equal( 'test-select' );
+      expect( multiselectDom.className ).to.equal( 'cf-multi-select active' );
+      expect( fieldset.getAttribute( 'aria-hidden' ) ).to.equal( 'false' );
+    } );
+
+    xit( 'should highlight the first item when keying down', function() {
+      multiselect.init();
+      var search = document.querySelector( '#test-select' );
+      search.click();
+      keyPress( search, 40 );
```

Comment:
Unfortunately that's for browser tests, not unit tests. I couldn't find anything included in mocha/jsdom